pkg/cfn/template/api_test.go

```diff
@@ -0,0 +1,72 @@
+package template
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("CloudFormation template", func() {
```

Comment:
These are basic tests for now, but in the future it would be nice to provide a high-level testing capability through custom gomega matchers.